### PR TITLE
DT-807 Use the HMPPS gradle plugin

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -1 +1,7 @@
+pluginManagement {
```
Comment (petergphillips) [inline settings.gradle.kts]: not needed since in gradle plugin portal anyway

File: dps-gradle-spring-boot-suppressions.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (petergphillips) [inline dps-gradle-spring-boot-suppressions.xml]: Did you mean to commit this file?  I was expecting it to be added to `.gitignore` and then have a
```
dependencyCheck {
  suppressionFiles = listOf()
}
```
block

Comment (mikehalmamoj) [reply dps-gradle-spring-boot-suppressions.xml]: Yeah probably best - I'll get rid.

**Review by petergphillips — DISMISSED** [review]
LGTM

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**